packages/components/src/internal/components/forms/QueryFormInputs.tsx

```diff
@@ -231,6 +233,7 @@ export class QueryFormInputs extends React.Component<QueryFormInputsProps, State
                             const multiple = col.isJunctionLookup();
                             const joinValues = multiple;
                             const id = col.fieldKey + i + (componentKey ?? '');
+                            const queryFilters = isAllSamplesSchema(col.lookup.schemaQuery) ? List([SAMPLES_WITH_TYPES_FILTER]) : undefined;
```

Comment:
what do you think about moving some of this into the `QueryLookup` class directly so that it is at least in one place? So this would become something like:
`const queryFilters = col.lookup.hasQueryFilters() ? List([col.lookup.queryFilters]) : undefined`

And then in the `QueryLookup` class add those methods for `hasQueryFilters and `get queryFilters` where the `isAllSamplesSchema` check and the `SAMPLES_WITH_TYPES_FILTER` lives within that one place?

That same type of change would then be applied in the `LookupCell.tsx` component.

Reply:
Done.